src/olympia/core/apps.py

```diff
@@ -1,14 +1,35 @@
 import logging
+import subprocess
 import warnings
 
 from django.apps import AppConfig
 from django.conf import settings
+from django.core.checks import Error, Tags, register
 from django.utils.translation import gettext_lazy as _
 
 
 log = logging.getLogger('z.startup')
 
 
+class CustomTags(Tags):
+    custom_setup = 'setup'
+
+
+@register(CustomTags.custom_setup)
```

Comment:
Just to clarify, this check is going to be executed when we run

```python
./manage.py check
```

Probably worth specifying that in the PR description as it is less than obvious, and I'm not 100% sure it is true even now.

Reply:
I added a docstring in https://github.com/mozilla/addons-server/pull/22044/commits/c8c6a3f816ffd124570395ba51d910f1b6a923f0 - the test I had added before should prove it's ran as part of `manage.py check`, which we trigger in the verify docker image GitHub Action. I've also made sure the check does fail if `uwsgi` isn't rebuilt.